staff/sergi-sanrama/precourse/tema3-functions/project3.js

```diff
@@ -0,0 +1,82 @@
+function bingo(){
+
+function userName(){ 
+  var person = prompt("Can you tell us your name?"); 
+    if (person) { 
+      return person;
+    } else { (person != typeof(string)) 
```

Comment:
wtf ?

Reply:
WHAT are you trying to resolve with that ```typeof```? do you understand ```typeof```? please, take a minute and RTFM => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof

Reply:
The type here does not make sense, fixed, thnks!